contracts/RibbonStakingRewards.sol

```diff
@@ -27,7 +27,7 @@ contract StakingRewards is
     IERC20 public stakingToken;
     uint256 public periodFinish = 0;
     uint256 public rewardRate = 0;
-    uint256 public rewardsDuration = 7 days;
+    uint256 public rewardsDuration = 30 days;
```

Comment:
I initially thought we would use setRewardsDuration(duration) during the setup steps, but hardcoding is easier.